sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyAlertConfigurationLiveTests.cs

```diff
@@ -24,9 +24,13 @@ public AnomalyAlertConfigurationLiveTests(bool isAsync) : base(isAsync)
         public async Task CreateAndGetAlertConfigurationWithWholeSeriesScope(bool useTokenCredential)
         {
             MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient(useTokenCredential);
+            await using DisposableDataFeed disposableDataFeed = await CreateTempDataFeedAsync(adminClient);
+            string metricId = disposableDataFeed.DataFeed.MetricIds[TempDataFeedMetricName];
+            await using DisposableDetectionConfiguration disposableDetectionConfig = await CreateTempDetectionConfigurationAsync(adminClient, metricId);
 
+            var detectionConfigId = disposableDetectionConfig.Configuration.Id;
             var scope = MetricAnomalyAlertScope.GetScopeForWholeSeries();
-            var metricAlertConfig = new MetricAnomalyAlertConfiguration(DetectionConfigurationId, scope);
+            var metricAlertConfig = new MetricAnomalyAlertConfiguration(detectionConfigId, scope);
```

Comment:
Instead of using a static data feed and a static detection config, we're now creating them on the fly.